### ISSUE TITLE
Moving more info to the Dashcam tab

### ISSUE THREAD

**Issue by joshkauffman** [Hivemapper/Community#197]

**Comment by jayalberts** [comment]
@joshkauffman do you want me to delete this issue?

**Comment by joshkauffman** [comment]
Yes please. I had clicked the wrong repo for it, and then it didn't let me do much as I didn't have access. I had deleted it, but I guess I didn't have access to _actually_ delete it. 
Thanks Jay
